src/hearings/containers/request-hearing/hearing-additional-instructions/hearing-additional-instructions.component.ts

```diff
@@ -49,7 +49,8 @@ export class HearingAdditionalInstructionsComponent extends RequestHearingPageFl
       ...this.hearingRequestMainModel,
       hearingDetails: {
         ...this.hearingRequestMainModel.hearingDetails,
-        listingComments: this.instructionsForm.value.instructions
+        listingComments: this.instructionsForm.value.instructions,
+        autolistFlag: (this.instructionsForm.value.instructions !== null) ? false : this.hearingRequestMainModel.hearingDetails.autolistFlag
```

Comment:
if this autolistFlag: (this.instructionsForm.value.instructions !== null) ? false : this.hearingRequestMainModel.hearingDetails.autolistFlag could be extracted to a new method and also add unit test for it?